### PR TITLE
Wac 1.18 docs updates

### PR DESCRIPTION
**Pull request by Emyrk**
- dev url support
- bitbucket support
- ability to specify filepaths other than coder.yaml (must be within .coder directory)

### CODE REVIEW

File: environments/workspaces-as-code/index.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ how to configure environments and their supporting infrastructure.
 
 ## Requirements
 
-- You must configure a [GitHub or GitLab OAuth service](../../admin/git.md)
+- You must configure a
+  [the Git OAuth service of your choice](../../admin/git.md)
```
Comment (vapurrmaid) [inline environments/workspaces-as-code/index.md:17]:
```suggestion
- You must configure a
  [Git OAuth service of your choice](../../admin/git.md)
```

File: environments/workspaces-as-code/index.md [hunk 2]
```diff
@@ -23,6 +24,26 @@ how to configure environments and their supporting infrastructure.
...
+## Creating a workspace using a template
+
+To create a new workspace using a template, go to **New Workspace** > **Create
+from Template**.
+
+![Create from template button](../../assets/create-from-template.png)
+
+When prompted, provide:
+
+- **Workspace Name**: A friendly name for your workspace
```
Comment (vapurrmaid) [inline environments/workspaces-as-code/index.md:36]: Suggestion: we don't mention `friendly` as it's corresponding to different IDs/representations at the data layer, which do not seem as beneficial in this context

Comment (Emyrk) [reply environments/workspaces-as-code/index.md:36]: I stole that language from the current environment form:
https://codercom-g9vx92axl-codercom.vercel.app/docs/environments/getting-started#2-create-an-environment

**Comment by Emyrk** [comment]
Waiting for the deployment to update so I can check my additions

**Review by khorne3 — APPROVED** [review]

**Comment by Emyrk** [comment]
@khorne3 is it alright if the deployment isn't coming up?
Can I merge this? Or do we wait until release?

I can let someone else click the button if that is the process :smile:

**Comment by khorne3** [comment]
@Emyrk Yeah, I think it only deploys when the base branch is master -- feel free to merge into 1.18

**Comment by Emyrk** [comment]
Ah ok. I'll merge now

**Pull request merged**